build-system/app-index/components/examples-document-mode-select.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-unused-vars */
+
+import {Component, h, render} from 'preact';
+
+export class ExamplesDocumentModeSelect extends Component {
+  render() {
```

Comment:
render's method signature `abstract render(props?:PropsType & ComponentProps<this>, state?:StateType, context?:any):JSX.Element|null;`

`props` and `state` are available as arguments, so no need to grab them from `this`.